examples/Migration/CampaignManagement/Step0.cs

```diff
@@ -0,0 +1,413 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Ads.AdWords.Lib;
+using System;
+using System.Collections.Generic;
+using System.Web;
+
+using aw = global::Google.Api.Ads.AdWords.v201809;
+
+namespace Google.Ads.GoogleAds.Examples.V1.Migration.CampaignManagement
+{
+    /// <summary>
+    /// This code example is the first one in a series of code examples that shows how to create
+    /// a Search campaign using AdWords API, and then migrate it to Google Ads API one
+    /// functionality at a time. See Step0.cs through Step5.cs for code examples in various
+    /// stages of migration. 
+    /// 
+    /// This code example represents the initial state, where AdWords API is used to create a
+    /// campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+    /// functionality has been migrated to Google Ads API.
+    /// </summary>
+    public class Step0 : ExampleBase
+    {
+        /// <summary>
+        /// Number of ads being added / updated in this code example.
+        /// </summary>
+        private const int NUMBER_OF_ADS = 5;
+
+        /// <summary>
+        /// The list of keywords being added in this code example.
+        /// </summary>
+        private readonly string[] KEYWORDS_TO_ADD = new string[]
+        {
+            "mars cruise",
+            "space hotel"
+        };
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example is the first one in a series of code examples that " +
```

Comment:
in this and subsequent files, "one" is superfluous, ie. "This code example is the first in a series..."

This applies to the comments above as well as the returned description.

Reply:
Done in all examples.